README.md

```diff
@@ -1,128 +1,105 @@
-# griptape-cli
+# Griptape Cloud CLI
 
 [![PyPI Version](https://img.shields.io/pypi/v/griptape-cli.svg)](https://pypi.python.org/pypi/griptape-cli)
 [![Tests](https://github.com/griptape-ai/griptape-cli/actions/workflows/tests.yml/badge.svg)](https://github.com/griptape-ai/griptape-cli/actions/workflows/tests.yml)
 [![Docs](https://readthedocs.org/projects/griptape/badge/)](https://griptape.readthedocs.io/)
 [![Griptape Discord](https://dcbadge.vercel.app/api/server/gnWRz88eym?compact=true&style=flat)](https://discord.gg/gnWRz88eym)
 
-CLI for the Griptape Framework and Cloud.
+The Griptape CLI is a command-line interface for interacting with features of [Griptape Cloud](https://www.griptape.ai/cloud).
+Today, it provides an emulator for Griptape Cloud Skatepark, which allows you to run and test your Structures locally. 
 
 ## Prerequisites
+    - A GitHub account
+    - Python 3.9 or later
 
-CLI `cloud` commands require a [Griptape Cloud](https://www.griptape.ai/griptape-cloud/) account.
-
-## Quick Start
+## Installation 
 
 1. Install griptape-cli
-   ```
-   pipx install griptape-cli
-   ```
-1. Verify installation
-   ```
-   gt --help
-   ```
-
-## Griptape Cloud
-
-### Skatepark Emulator
+    ```bash
+    pipx install griptape-cli
+    ```
+
+   You can also install a pre-release version from GitHub.
+    ```bash
+    git clone https://github.com/griptape-ai/griptape-cli.git
+    ```
+
+    ```bash
+    pipx install ./griptape-cli
+    ```
+2. Verify installation
+    ```bash
+    gt --help
+    ```
+
+## Skatepark Emulator
 You can use the CLI to spin up a local emulator of the Griptape Cloud Skatepark. This is useful for testing and development.
```

Comment:
This is nuanced, but it sounds like the emulator is Skatepark (and I like that, btw), but nowhere are we calling the cloud offering "Griptape Cloud Skatepark" AFAIK. My recommendation:
1. The emulator is called Skatepark
2. It emulates Griptape Cloud's Managed Structures
3. It's not "useful for testing and development," but rather we should frame this as "Use the Skatepark emulator to develop, test, and validate that your program will operate as expected when deployed as a Griptape Cloud Manage Structure."